### DISCUSSION TITLE
Change of font

### DISCUSSION THREAD

**Discussion by Ryanbeck144** [KadenKercheval/group7#7] [Ideas]
Thoughts on changing the font for the recipes for a more appealing look?

**Comment by KadenKercheval** [comment]
I feel like that is a good idea. Also, the color of the text on the carousel is blending in with the photo. I feel like we could also change that to a black or more contrasting color depending on the image.

**Reply by Ryanbeck144** [reply]
Sounds great I'll look into it here shortly

**Reply by Ryanbeck144** [reply]
I changed the font weight and color to a grey but it's hard to find a color that looks good on all images, lmk what yall think or if i should change it back to white.

**Comment by MSM7437** [comment]
So for the index page, the text on top of the images doesn't all have to be the same does it? I submitted a pull request changing the grey text of "marry me chicken" back to white, but if yall wanna keep it grey we can leave it.

**Comment by MSM7437** [comment]
A different font could look good tho
